### PR TITLE
Add package power-grid-model

### PR DESCRIPTION
**Pull request by TonyXiang8787**
This is a PR to include `power-grid-model` into `conda-forge`.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by TonyXiang8787** [comment]
@mgovers @nitbharambe @petersalemink95 please confirm in this PR that  you would like to be the maintainer of `power-grid-model` in `conda-forge`.

**Comment by TonyXiang8787** [comment]
Dear @conda-forge/help-python-c, I see build errors in all platforms about finding `eigen` headers. If I put `eigen` as build dependency in `meta.yaml`. How can I in `setup.py` find the include path of the `eigen` header files?

**Comment by mgovers** [comment]
> 

you probably need to use a bot to ping: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-conda-forge-team

**Comment by TonyXiang8787** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c

**Comment by nitbharambe** [comment]
> @mgovers @nitbharambe @petersalemink95 please confirm in this PR that you would like to be the maintainer of `power-grid-model` in `conda-forge`.

Confirming I would like to be a maintainer of `power-grid-model`.

**Comment by petersalemink95** [comment]
> @mgovers @nitbharambe @petersalemink95 please confirm in this PR that you would like to be the maintainer of `power-grid-model` in `conda-forge`.

I would like to be a maintainer of `power-grid-model` in `conda-forge`

**Comment by TonyXiang8787** [comment]
@conda-forge/help-python-c, ready for review!

File: recipes/power-grid-model/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-{% set version = "1.4.69" %}
+{% set version = "1.4.70" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/power-grid-model-{{ version }}.tar.gz
-  sha256: 40ba3f8e98b2e883f8f8015064c13fa2f148141c778df2511c4a952ecede07de
```
Comment (mgovers) [inline recipes/power-grid-model/meta.yaml]: this sounds like it's going to be a hassle to keep up to date. how can this be automated?

Comment (TonyXiang8787) [reply recipes/power-grid-model/meta.yaml]: There will be bot later which will update the version and hash automatically.

**Pull request merged**